jekyll/_data/sidenav.yml

```diff
@@ -30,8 +32,10 @@ en:
             link: config-editor
       - name: VCS integration
         children:
-          - name: GitHub integration overview
+          - name: GitHub OAuth integration overview
             link: github-integration
+          - name: GitHub apps integration overview
```

Comment:
```suggestion
          - name: GitHub Apps integration overview
```

Reply:
@rosieyohannan I went through the PR and changed a lot of the "GitHub Apps" instances to "GitHub App" or "the CircleCI GitHub App"... shall we do that for the sidenav as well?